applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -189,24 +189,45 @@ For an example of a bootloader Kconfig configuration file defined by the applica
   Both mentioned firmware upgrade methods are not used simultaneously by any of the configurations.
   For example, the ``nrf52840dk/nrf52840`` board in ``mcuboot_smp`` file suffix uses only the background DFU and does not enable the serial recovery feature.
 
-MCUboot bootloader on nRF54L
-----------------------------
+MCUboot bootloader on nRF54
+---------------------------
 
-The nRF54L SoC Series enhances security and reduces boot times by using hardware cryptography in the MCUboot immutable bootloader.
-The |NCS| allows using hardware cryptography for ED25519 signature (:kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519`) on the nRF54L SoC Series.
+The nRF54 SoC Series enhances security and reduces boot times by using hardware cryptography in the MCUboot immutable bootloader.
+The |NCS| allows using hardware cryptography for ED25519 signature (:kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519`) on the nRF54 SoC Series.
 
-You can enhance security further by enabling the following sysbuild Kconfig options:
+You can enhance security further by enabling the :kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_TYPE_PURE` sysbuild Kconfig option.
+This option enables using a pure signature of the image, verifying signature directly on image, rather than on its hash.
+However, you cannot use this option if the secondary image slot uses external memory.
```

Comment:
`However, you cannot use this option if the secondary image slot uses external memory.` -> Does the limitation apply also to nRF54H?

Reply:
I think so. @tomchy, could you confirm?